cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/TranslationContext.kt

```diff
@@ -54,4 +54,9 @@ class TranslationContext(
      * the [TranslationResult.finalCtx] this may either be null or the last component analyzed.
      */
     var currentComponent: Component? = null,
+
+    // TODO: move somewhere better or maybe make translation context the statistics holder instead
+    // of the result?
+    var inferredFunctions: Int = 0,
```

Comment:
not needed because we can just count inferred nodes at the end, should be removed